### ISSUE TITLE
XsollaAuth.AuthViaDeviceID sometimes fail

### ISSUE THREAD

**Issue by artyom-v** [xsolla/store-unity-sdk#68]
XsollaAuth.AuthViaDeviceID method sometimes failed on Android devices with errors like:
1. "body.device_id in body should be at least 16 chars long"
2. "body.device in body should be at most 100 chars long"

First error occurred because DeviceIdUtil.GetDeviceId() can return device id without leading zeroes [link](https://stackoverflow.com/a/29890124)
Fix:
`DeviceIdUtil.GetDeviceId().PadLeft( 16, '0' )`

Example of data for second error:
```
DeviceModel: TECNO MOBILE LIMITED TECNO KG5m
DeviceName: IzJZRVYtED8AAAAAAAAAggFURUNOTyBLRzVtICg2NjQpAEFnVTRNek13QUt3d0FBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUFBQUE
DeviceId: 5a08660fbafe3a8f
```
SDK code:
```
var requestData = new AuthViaDeviceIdRequest {
	device = $"{deviceInfo.DeviceModel}:{deviceInfo.DeviceName}",
	device_id = deviceInfo.DeviceId
};
```
So final length for device is > 100
Extra characters should be cut IMO.

**Comment by DenisRakhimovXsolla** [comment]
Hey @artyom-v !
Thanks for the report!
Will be fixed in the next release

**Comment by DenisRakhimovXsolla** [comment]
Fixed in version 2.1.0
